akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -923,7 +923,7 @@ private[remote] object ArteryTransport {
 
   val ProtocolName = "artery"
 
-  val Version = 0
+  val Version: Byte = 0
```

Comment:
No longer a compile-time constant. Sadly, I don't know a way to specify a byte literal. Sigh.